CalibCalorimetry/EcalLaserSorting/interface/LaserSorter.h

```diff
@@ -123,7 +123,7 @@ class LaserSorter : public edm::EDAnalyzer {
   void analyze(const edm::Event&, const edm::EventSetup&) override;
   void endJob() override;
   void beginJob() override;
-  void beginRun(edm::Run const&, edm::EventSetup const&) override;
+  void beginRun(edm::Run const&, edm::EventSetup const&);
```

Comment:
As I just explained in the other comment you need to make a `edm::one::EDAnalyzer<edm::one::WatchRuns>` if you want to use `beginRun()`.

Reply:
Implemented